woocommerce/woocommerce-ios
# Hub Menu: manage review push notification

Part of #5509 

### Description
Previously the Review Detail was showed from the Review Tab when a push notification was coming. As part of the implementation of the Hub Menu, I implemented the flow for showing a review detail from the Hub Menu, since we need to present a detail when a push notification is coming in background.


### Testing instructions
1. Run the app on your device (also in debug, push notifications now are working properly also on `develop` without configuring a sandbox environment). 
2. Close the app or put the app in foreground.
3. Try to receive a push notification for a new review, and tap it.
4. The review detail should be presented from the hub menu.


---
